monkey/monkey_island/cc/services/agent_signals_service.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+from monkey_island.cc.models import Agent, AgentSignals
+from monkey_island.cc.repository import ISimulationRepository
+
+
+class AgentSignalsService:
+    def __init__(self, simulation_repository: ISimulationRepository):
+        self._simulation_repository = simulation_repository
+
+    def get_signals(self, agent: Agent) -> AgentSignals:
+        """
+        Gets signals from the agent
+
+        :param agent: The `Agent` from which to retrieve the signals
+        :return: `AgentSignals` model from the corresponding agent
+        """
```

Comment:
I don't think we should mention data types in docstrings.
```suggestion
        """
        Gets the signals sent to a particular agent

        :param agent: The agent whose signals need to be retrieved
        :return: Signals sent to the relevant agent
        """
```